### PR TITLE
Use `bool` for control bits instead of `Field2`

### PR DESCRIPTION
**Pull request by cjpatton**
Stacked on #68.

This makes the code a bit clearer.

### CODE REVIEW

File: poc/tests/test_vidpf.py [hunk 1]
```diff
@@ -35,16 +35,16 @@ def test_eval_invariants(self):
             # The control bits should be secret shares of one, i.e., the
             # control bit should be set for one and only one of the
             # aggregators.
-            self.assertEqual(node[0].ctrl + node[1].ctrl, Field2(1))
+            self.assertTrue(node[0].ctrl ^ node[1].ctrl)
```
Comment (jimouris) [inline poc/tests/test_vidpf.py:38]: maybe `node[0].ctrl != node[1].ctrl` for clarity?

File: poc/tests/test_vidpf.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ def test_eval_invariants(self):
 
             # The control bits should be secret shares of zero, i.e., either
             # both have the bit set or neither does.
-            self.assertEqual(node[0].ctrl + node[1].ctrl, Field2(0))
+            self.assertFalse(node[0].ctrl ^ node[1].ctrl)
```
Comment (jimouris) [inline poc/tests/test_vidpf.py:64]: assert `node[0].ctrl == node[1].ctrl`

File: poc/tests/test_vidpf.py [hunk 3]
```diff
@@ -247,7 +247,7 @@ def test_malformed_correction_word_ctrl(self):
         malformed_level = randrange(vidpf.BITS)
         (seed_cw, ctrl_cw, w_cw, proof_cw) = public_share[malformed_level]
         malformed = ctrl_cw.copy()
-        malformed[randrange(2)] += Field2(1)
+        malformed[randrange(2)] ^= True
```
Comment (jimouris) [inline poc/tests/test_vidpf.py:250]: Can we use the Boolean `not` instead of XOR here?

**Review by jimouris — APPROVED** [review]

File: poc/tests/test_vidpf.py (hunk 1 above)
Comment (cjpatton) [reply poc/tests/test_vidpf.py:38]: Done.

File: poc/tests/test_vidpf.py (hunk 2 above)
Comment (cjpatton) [reply poc/tests/test_vidpf.py:64]: Done.

File: poc/tests/test_vidpf.py (hunk 3 above)
Comment (cjpatton) [reply poc/tests/test_vidpf.py:250]: Done.

**Pull request merged**